lib/fake_twilio.rb

```diff
@@ -0,0 +1,20 @@
+class FakeTwilio
```

Comment:
Love that you mocked Twilio to avoid sending texts! It's definitely worthwhile using irb or a throwaway Ruby file to figure out how this code works for your own learning.